### ISSUE TITLE
36MB repo...

### ISSUE THREAD

**Issue by FarahZaqout** [FACG5/w7-blogger#19]
![36mb](https://user-images.githubusercontent.com/25854070/44162946-7e716a00-a0ca-11e8-904c-2e856fb3e498.png)

That's way too much space for a simple 2 day project.

7aram 3leeko.

**Comment by ali-7** [comment]
yes, i notice that but i don't know from where this huge size is ? 
we only have an img with 2.2mb size and the rest is the files with bytes !

**Comment by FarahZaqout** [comment]
Yeah I noticed that the images are only 2mb, but 2mb is still a lot for one file, use urls instead of actual images. Anyway we'll see if someone knows how to figure out what's making the repo so large.
